.pipelines/.vsts-vhd-builder-release-windows.yaml

```diff
@@ -0,0 +1,54 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)_$(Build.SourceBranchName)_$(BuildID)
+trigger: none
+
+parameters:
+- name: build2019
+  displayName: Build 2019
+  type: boolean
+  default: true
+- name: build2019containerd
+  displayName: Build 2019 containerd
+  type: boolean
+  default: true
+- name: dryrun
+  displayName: Dry run
+  type: boolean
+  default: false
+
+variables:
+  CONTAINER_IMAGE:  'mcr.microsoft.com/oss/azcu/go-dev:v1.28.5'
+
+phases:
+  - phase: build_vhd_2019
+    condition: eq('${{ parameters.build2019 }}', true)
+    queue:
+      name: Hosted Ubuntu 1604
+      timeoutInMinutes: 180
+    steps:
+      - bash: |
+          echo '##vso[task.setvariable variable=DRY_RUN]${{parameters.dryrun}}'
+          echo '##vso[task.setvariable variable=HYPERV_GENERATION]V1'
+          echo '##vso[task.setvariable variable=AZURE_VM_SIZE]Standard_D4s_v3'
+          echo '##vso[task.setvariable variable=CONTAINER_RUNTIME]docker'
+          echo '##vso[task.setvariable variable=WINDOWS_SKU]2019'
+        displayName: Setup Build Variables
+      - template: ./templates/.builder-release-template-windows.yaml
+        parameters:
+          artifactName: 2019
+
+  - phase: build_vhd_2019_containerd
```

Comment:
Just a reminder: we don't need to cache so many k8s versions for containerD case.